### ISSUE TITLE
SCM link is incorrect

### ISSUE THREAD

**Issue by barsnick** [fedora-infra/fedora-packages#299]
On every package's page, the link to its git repo, titled "SCM", is currently incorrect. Or it's the redirect it points to which is incorrect.

E.g., keepass's page
https://apps.fedoraproject.org/packages/keepass
links to
https://pkgs.fedoraproject.org/cgit/keepass.git
which redirects to
https://src.fedoraproject.org/cgit/keepass.git
but should be linking to
https://src.fedoraproject.org/cgit/rpms/keepass.git

Since apps.fedoraproject.org is the one-stop shop for ALL info about a package, every link needs to be correct. ;-)

**Comment by pypingou** [comment]
This is a duplicate of https://github.com/fedora-infra/fedora-packages/issues/294

Thanks for your report though :)

**Comment by barsnick** [comment]
Well d'uh, thanks. I didn't check for closed bugs, as the issue seemed "open". ;-) When will the fix be deployed?

**Comment by pypingou** [comment]
@cverna is working on updating fedora-packages :)

**Comment by techtonik** [comment]
Still no progress. Anything we can help?

**Comment by cverna** [comment]
@techtonik I am still working on it, the application development is I guess done. I am now working on the deployment and I am running in some dependency hell 🤒

**Comment by techtonik** [comment]
@cverna are there any issues to track? Lurking in the dark is not something one can learn from.

**Comment by cverna** [comment]
@techtonik for now you can track the PR to update TurboGears2 to latest upstream https://src.fedoraproject.org/rpms/TurboGears2/pull-request/1

Once this is merged I ll be able to build fedora-packages and then deploy it

**Comment by techtonik** [comment]
No response on that PR in 5 days and no info if https://src.fedoraproject.org/user/lmacken is active.

**Comment by techtonik** [comment]
@cverna https://src.fedoraproject.org/rpms/TurboGears2/pull-request/1is merged.

**Comment by cverna** [comment]
@techtonik, there is a instance deployed in [stg](https://apps.stg.fedoraproject.org/packages) if you would like to look at it.

I am preparing a release and I hope to deploy it soon in production.

**Comment by cverna** [comment]
@techtonik the new version of packages was deployed today.

**Comment by techtonik** [comment]
I now can not find the link. :D

**Comment by techtonik** [comment]
https://apps.fedoraproject.org/packages/keepass/ sources tab doesn't function.

**Comment by pypingou** [comment]
https://apps.fedoraproject.org/packages/_w/package_sources?subpackage_of=&package_name=keepass returns a 500, something to investigate in January
